pkgs/tools/audio/goxlr-utility/default.nix

```diff
@@ -5,34 +5,35 @@
 , pkg-config
 , libpulseaudio
 , dbus
-, speechd-minimal
+, openssl
+, speechd
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "goxlr-utility";
-  version = "1.1.1";
+  version = "2024-08-06";
 
   src = fetchFromGitHub {
     owner = "GoXLR-on-Linux";
     repo = "goxlr-utility";
-    # v1.1.1 was released with broken Cargo.lock so we'll use later commit where it was fixed
-    rev = "26a818366e7f28802592baa463bb57fc9eccbe27";
-    hash = "sha256-tUAZSfoC9bp7gK884nVGumtcLb2LAw+zQRSoVS8r+QI=";
+    rev = "dcd4454a2634f5a2af10f00c1cbcb016241ce2cb";
+    hash = "sha256-kWfCFsk0GhqX+pYOTeJd7XHlcWOX4D6fmIU/4nylU3Y=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
       "ksni-0.2.1" = "sha256-cq3PAqkiYEv4MW5CtT7eau38Mf4uxdJ1C2fw640RXzI=";
-      "tasklist-0.2.13" = "sha256-DMaVOo1TSIyjspybok1y07oNxGjHaPSC6qe4NmDfNgE=";
-      "xpc-connection-sys-0.1.1" = "sha256-bzxzzTwPwa7flt8Jm9OcoBLwp3zn/V5WS2hTZjXV1/M=";
+      "tasklist-0.2.15" = "sha256-YVAXqXuE4azxYi0ObOq4c9ZeMKFa2KjwwjjQlAeIPro=";
+      "xpc-connection-sys-0.1.1" = "sha256-VYZyf271sDjnvgIv4iDA6bcPt9dm4Tp8rRxr682iWwU=";
     };
   };
 
   buildInputs = [
     libpulseaudio
     dbus
-    speechd-minimal
+    speechd
```

Comment:
Why was the version of speechd changed to the fat one?

Reply:
Was thinking about adding a comment about it, but it was only to make it easier to build on NixOS 24.05 as it doesn't have `speechd-minimal`. It still does require using rust from unstable so probably not worth doing the change as it can't be merged to 24.05 anyways.